acceptance/tests/facts/verify_facts_match_schema.rb

```diff
@@ -0,0 +1,111 @@
+test_name 'C62454: verify facts return from facter match the schema' do
```

Comment:
I haven't started looking at unit tests yet. Its next on my list after I finish updating the last 13 acceptance tests.

After looking at the Unit test, yeah we can probably lose this test at the acceptance level. I'll remove it.